docs/_data/prague-2021-schedule.yaml

```diff
@@ -0,0 +1,96 @@
+writing_day:
+- time: '10:00'
+  duration: '00:30'
+  title: 'Platform opens'
+- duration: '0:15'
+  title: Morning introduction - Introduction to Sprints & Project Introductions
+- duration: '2:15'
+  title: Team up and start working
+- duration: '1:00'
+  title: Lunch Break (60 mins)
+- time: '17:00'
+  title: End of Writing Day
+
+talks_day1:
+- time: '9:00'
+  duration: '1:00'
+  title: Platform opens
+- duration: '0:45'
+  title: Write the Docs Team - Introduction to Write the Docs
+- title: <b>Unconference Starts</b>
+- duration: '0:40'
+  slug: hitchhiker-s-guide-to-documentation-tools-and-processes-lukas-reu-ner
+- duration: '0:10'
+  title: Break (10 mins)
+- duration: '0:40'
+  slug: more-than-words-reviewing-and-updating-your-information-architecture-lana-brindley-she-her
+- duration: '0:5'
+  title: Switch speakers (5 mins)
+- duration: '0:40'
+  slug: so-you-need-to-give-bad-news-to-users-ryan-macklin-he-they
+- duration: '1:00'
+  title: Lunch (60 mins)
+- duration: '0:30'
+  title: Lightning talks (30 mins)
+- duration: '0:5'
+  title: Switch speakers (5 mins)
+- duration: '0:40'
+  slug: when-documenting-is-designing-how-to-assist-api-design-as-a-technical-writer-fabrizio-ferri-benedetti
+- duration: '0:5'
+  title: Switch speakers (5 mins)
+- duration: '0:40'
+  slug: adventures-in-setting-up-a-knowledge-system-for-a-research-group-victor-sluiter
+- duration: '0:10'
+  title: Break (10 mins)
+- duration: '0:40'
+  slug: alchemy-in-adversity-how-to-become-better-and-more-resilient-tech-writers-in-a-turbulent-work-environment-anna-korinna-n-meth-szab
+- duration: '0:5'
+  title: Switch speakers (5 mins)
+- duration: '0:40'
+  slug: customer-feedback-is-the-fuel-in-our-engine-kevin-hwang
+- duration: '0:25'
+  title: Day 1 wraps up
+- time: '18:00'
+  duration: '2:00'
+  title: <b>Social Event</b>
+- title: <b>Social Event Ends</b>
+
```

Comment:
I can't figure out how to comment on a different part of the file (I blame Fridays) but there's a missing bit in the location of the Social Event after the Monday schedule. we can just write "Spatial Chat, details will be posted during the conference" or something generic like that :)